### ISSUE TITLE
exclude option not working?

### ISSUE THREAD

**Issue by Andre601** [ojacques/mkdocs-git-committers-plugin-2#43]
Encountered a strange behaviour of the plugin with its exclude option.

For visual reasons (content being added after `{{ page.content }}` do I want to exclude `docs/changelog/index.md` from having committer info displayed.

However, no matter if I use `docs/changelog/index.md` or `changelog/index.md`, the page still displays the committers.

Am I doing something wrong? Or is just the path not properly checked against the root or docs dir of the project?
Or could it be caused by the added content making the plugin assume the page received changes, which somehow ignores the exclude list?

The plugin mentions fetching data for the page:
```
INFO    - git-committers: fetching contributors for docs/changelog/index.md
```

Also, I want to mention it also fetches contributors for certain pages, even tho the readme mentions it would only do so if the pages received changes since last caching. These pages didn't change at all, so I assume the plugin simply doesn't save to cache properly?

**Comment by ojacques** [comment]
@Andre601 - can you check which version of the plugin you use? Also, the cache (`.cache/plugin/git-committers/page-authors.json `) may contain the authors if exclude was added after the cache got populated.

**Comment by Andre601** [comment]
```sh
$ pip show mkdocs-git-committers-plugin-2
Name: mkdocs-git-committers-plugin-2
Version: 2.2.2
Summary: An MkDocs plugin to create a list of contributors on the page. The git-committers plugin will seed the template context with a list of GitHub or GitLab committers and other useful GIT info such as last modified date
Home-page: https://github.com/ojacques/mkdocs-git-committers-plugin-2/
Author: Byrne Reese, Olivier Jacques
Author-email: byrne@majordojo.com, ojacques2@gmail.com
License: MIT
Location: C:\Users\Andreas\AppData\Local\Programs\Python\Python311\Lib\site-packages
Requires: gitpython, mkdocs, requests
Required-by:
```

Also, I tried with the cache file deleted before and it still created the docs...
I tried with `docs/changelog/index.md` - which the plugin mentions when fetching committers - and with `changelog/index.md`, but neither does work.

Also, I want to point out that the plugin is still generating a cache file, even when it is not enabled, and reports it as such (`INFO    -  git-committers plugin DISABLED`), which is strange for me, as the plugin should imo do literally nothing while disabled... else this option makes no sense to begin with.

**Comment by ojacques** [comment]
Fixed in [2.2.3](https://github.com/ojacques/mkdocs-git-committers-plugin-2/releases/tag/2.2.3)
